Various technical notes/transition_infos.json

```diff
@@ -825,15 +803,9 @@
     },
     {
       "area_id": "0x5511C46C",
-      "area_name": "Apu Illapu Shrine (Spinjas)",
+      "area_name": "Apu Illapu Shrine",
```

Comment:
Humm, I'm partial to keeping "spinjas" in the name just because no one calls it apu illapu XD but not a blocker.

Reply:
Yeah I don't agree. I think it's stupid to include '(Spinjas)' in the offical area_name.

Reply:
Given my view on what this file represents, adding "spinjas" as a nice info for the player on the HUD and logs would be more an in-code thing.

> in the offical area_name.

So agreed.